### DISCUSSION TITLE
Svelte + Typescript + Snopack + Prisma => Unhandled Runtime Error?

### DISCUSSION THREAD

**Discussion by arda-** [prisma/prisma#5382] [Q&A] [answered]
:wave: Hello!

Importing Prisma gives me an error. My stack is a combination of Svelte + Typescript + Snowpack + Prisma. Got my schema generated and types imported, but I can't actually use the Prisma client:

App.svelte:
```svelte
<script lang="ts">
import { PrismaClient } from '@prisma/client'
const prisma = new PrismaClient();
</script>
```

gives me the following error:

```
1 of 1 unhandled error

Unhandled Runtime Error
Script error.
Source
No source file.
```

Searching the Discussions didn't yield me similar results; any guidance would be appreciated. Thanks!

**Comment by ryands17** [comment, marked as answer]
@arda- 👋 

Currently Prisma only works on the server side of things, so you can for now only use it with Node, or a SSR framework like Next.js.

**Reply by arda-** [reply]
Aha, that makes sense. Thanks Ryan!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
